### PR TITLE
Rename setServerUrl to setLogUrl

### PR DESCRIPTION
**Pull request by guperrot**

### CODE REVIEW

**Comment by guperrot** [comment]
This is for develop. After this is merged I will need to resolve conflicts in updates branch...

File: apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/MainActivity.java [hunk 1]
```diff
@@ -43,9 +45,19 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        if (serverUrl != null) {
-            MobileCenter.setServerUrl(serverUrl);
+        String logUrl = sSharedPreferences.getString(LOG_URL_KEY, null);
+        if (logUrl != null) {
+            try {
+                Method setLogUrl;
+                try {
+                    setLogUrl = MobileCenter.class.getMethod("setLogUrl", String.class);
+                } catch (NoSuchMethodException e) {
+                    setLogUrl = MobileCenter.class.getMethod("setServerUrl", String.class);
```
Comment (jaelim-ms) [inline apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/MainActivity.java]: I know this is a test application but why don't you put a comment why you are doing it here. I know why and you know why but others don't know. 😄

File: sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/MobileCenter.java [hunk 2]
```diff
@@ -47,9 +47,9 @@
     private boolean mLogLevelConfigured;
 
     /**
-     * Custom server Url if any.
+     * Custom log url if any.
```
Comment (jaelim-ms) [inline sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/MobileCenter.java]: Let's use all upper cases for `url` for comments and docs.

File: apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/MainActivity.java (hunk 1 above)
Comment (guperrot) [inline apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/MainActivity.java]: @jaelim-ms added comment

**Pull request merged**